### PR TITLE
Add group and component ordering.

### PR DESCRIPTION
**Pull request by glogiotatidis**
Components with lower `order` values are listed first.

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@jpetto r?

File: js/statusboard.js [hunk 1]
```diff
@@ -94,46 +108,77 @@ var ComponentGroup = React.createClass({
...
             else {
-                groups.default.push(component);
+                groups.default.components.push(component);
             }
-
         });
+
+        for (var property in groups) {
+            if (groups.hasOwnProperty(property)) {
+                if (property == 'default')
```
Comment (jpetto) [inline js/statusboard.js]: Make the linters happy and use `===`?

File: js/statusboard.js [hunk 2]
```diff
@@ -94,46 +108,77 @@ var ComponentGroup = React.createClass({
...
+                    // Default group has already calculated order.
+                    continue;
+                groups[property].order = this.getGroupOrder(groups[property].components);
+                console.log(property, groups[property].order);
```
Comment (jpetto) [inline js/statusboard.js]: Clean up the `console.log` here and below?

File: js/statusboard.js [hunk 3]
```diff
@@ -94,46 +108,77 @@ var ComponentGroup = React.createClass({
...
+            }
+        };
+
         return groups;
     },
+    getGroupOrder: function(components) {
```
Comment (jpetto) [inline js/statusboard.js:153]: Maybe add a comment describing the purpose & logic of this function?

**Comment by jpetto** [comment]
A couple of small nit-picks, but no big complaints. r+wc

**Comment by glogiotatidis** [comment]
Thanks John!

**Pull request merged**